package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sealink-ecom-engine-css",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "SeaLink Ecom Engine shared styles",
   "main": "index.js",
```

Comment:
There is no `index.js` in the repo.

Reply:
@dwaring I've changed it to sass/ecom.sass